README.md

```diff
@@ -8,6 +8,39 @@ effectively together with [nginx's autoindex module](http://nginx.org/en/docs/ht
 
 A sample nginx configuration is also included which mounts **file browser** under root (`/`) and mounts files to be listed under `/files` path. Hence is the `filesBaseUrl` under
 
+## Installation / Configuration
+
+The system has a default to use the location's root (E.G. the site's root location `www.example.net/`, and the physical of `/opt/www/`). This can be configured to move both the ***installation*** location and the location to be browseable.
+
+1. Clone the repo to the server
+2. Create an installation location (E.G. `/var/www/files/`)
+3. Copy the `css`, `image`, `js` directories to the installation location
+4. Copy the `index.html` to the installation location
```

Comment:
up until here, we can just do it in one step of: "clone the repo under `/var/www/files/` or any directory you want to install the file browser's web UI from". WDYT?

also I would argue that choosing directory name `files/` for web UI, which in default setup is used for the listed files is a confusing choice. what about something like `/var/www/ui/`?